Update Step Functions troubleshooting doc for missing TaskScheduled event (#23567)
DataDog/documentation: content/en/serverless/step_functions/troubleshooting.md

- Verify that you can see both Lambda traces and Step Function traces in Datadog.

> - Verify that you are using Python layer v75+ or Node.js layer v94+.

Reviewer: @kimi-p  Should me also update the Python and Node.js layer version for the 128 bits change?

Author: This is done in the previous PR https://github.com/DataDog/documentation/pull/23546

- In your AWS console, open your Step Function and ensure that your state machine has `"Payload.$": "States.JsonMerge($$, $, false)"` on the Lambda steps.

> - Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][4]. If you do not have `TaskScheduled` event and only have `LambdaFunctionScheduled` event, please update the task in Step Functions definition to use the recommended Lambda integration by following the [AWS documentation][5].

Reviewer 2: ```suggestion
- Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][4]. If you do not have a `TaskScheduled` event and only have a `LambdaFunctionScheduled` event, update the task in Step Functions definition to use the recommended Lambda integration. See the [AWS documentation][5] for instructions on how to do this.
```
